suite/tests/CMakeLists.txt

```diff
@@ -2532,7 +2532,7 @@ torunonly_ci(client.blackbox ${ci_shared_app} client.blackbox.dll.A
   "" "" "")
 
 tobuild_ci(client.option_parse client-interface/option_parse.cpp
```

Comment:
Same as what you did for -x (set it to 3 then override to 4), maybe do the same for z (use -no_z first, then override to true using -z_alias)

Reply:
-z is a string: do you mean to add an alias for a bool option (like `-flag`)?

Reply:
Oh z is not a bool. Maybe do it for some bool option too

Reply:
I'll assume that's what you meant.  Adding 2 aliases so will test 3 names.